docs/src/modules/components/AppNavDrawer.js

```diff
@@ -480,13 +462,17 @@ function AppNavDrawer(props) {
             onClick={(event) => {
               setAnchorEl(event.currentTarget);
             }}
-            endIcon={<ArrowDropDownRoundedIcon fontSize="small" sx={{ ml: -0.5 }} />}
+            endIcon={
```

Comment:
**Off-topic** I have noticed this behavior for Windows users, I'm not sure how to solve this layout constraint issue:

https://user-images.githubusercontent.com/3165635/155136315-3cd87198-1e5a-4f06-8fa7-084b557ab40f.mp4

Reply:
I can stretch the space a little bit more.

Reply:
<img width="279" alt="Screen Shot 2565-02-23 at 09 38 18" src="https://user-images.githubusercontent.com/18292247/155253117-7e3a8f92-d52e-49ab-b409-c2938c47ba20.png">

I have to remove`.0.0`, otherwise I can't think of a way to reduce the space.